src/main/java/friday/Task.java

```diff
@@ -0,0 +1,51 @@
+package friday;
+
+/**
+ * Represents a task in the Friday application.
+ */
+public class Task {
+    protected String name;
+    protected boolean isDone;
+
+    public Task(String name) {
+        this.name = name;
+        this.isDone = false;
+    }
+
+    /**
+     * Returns a status icon based on whether the task is done.
+     * @return A string representing the status icon.
+     */
+    public String getStatusIcon() {
+        return (isDone ? "X" : " ");
+    }
+
+    /**
+     * Mark the task as done.
+     */
+    public void mark() {
+        this.isDone = true;
+    }
+
+    /**
+     * Mark the task as not done.
+     */
+    public void unmark() {
+        this.isDone = false;
+    }
+
+    @Override
+    public String toString() {
+        return "[" + getStatusIcon() + "] " + name;
+    }
+
+    /**
+     * Checks if keyword is contained in task description.
+     * @param keyWord The keyword to check for.
```

Comment:
Should the Javadocs have space between the comments and the @ tags?

```suggestion
     * Checks if keyword is contained in task description.
     *
     * @param keyWord The keyword to check for.
```